app/ui.js

```diff
@@ -98,6 +98,8 @@ var UI;
         lastKeyboardinput: null,
         defaultKeyboardinputLen: 100,
 
+        reconnect_enabled: false,
```

Comment:
That's what I thought, but I didn't come up with a good name. Any suggestions? `last_connect_successful`?